ports/nrf/boards/wt51822-s4at/mpconfigboard.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Glenn Ruben Bakke
```

Comment:
If you want you can use your name here. Also changing the year to 2017 as file is new.